components/director/internal/domain/formation/service.go

```diff
@@ -307,7 +327,26 @@ func (s *service) DeleteFormation(ctx context.Context, tnt string, formation mod
 		return nil, errors.Wrapf(err, "While enforcing constraints for target operation %q and constraint type %q", model.DeleteFormationOperation, model.PreOperation)
 	}
 
+	formationID := ft.formation.ID
 	formationName := ft.formation.Name
+	log.C(ctx).Infof("Updating formation with ID: %q and name: %q to state: %s", formationID, formationName, model.DeletingFormationState)
```

Comment:
Isn't `DELETING` state just for async webhooks in the future? Synchronous webhooks and regular formations just jump from `READY` to `CREATE_ERROR` after creation or remain in `READY` or from `READY` to `DELETE_ERROR` when deleting. The `DELETING` and `INITIAL` states are for when we are waiting for the response on async webhooks.
Otherwise we will not be able to not differentiate when we are in the async case based on the state of the formation.